### ISSUE TITLE
Migrations cant be executed due to collation

### ISSUE THREAD

**Issue by htuscher** [daniellienert/dl-assetsync#7]
`DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci` doesn't work if you are on Neos 4:

```
Field type or character set for column 'resource' does not mach referenced column 'persistence_object_identifier'
```

**Comment by daniellienert** [comment]
Hey Hans, 
Thanks for reporting.
Seems I never installed the package on a 4.x Neos. Updating projects with the package installed from 3.x to 4.x worked fine.
I hopefully fixed it in master and will release a new version asap.

**Comment by daniellienert** [comment]
Fixed with Version 2.2.0
